cmd/osm-controller/osm-controller.go

```diff
@@ -83,6 +85,7 @@ func init() {
 	flags.StringVar(&osmNamespace, "osm-namespace", "", "Namespace to which OSM belongs to.")
 	flags.StringVar(&webhookConfigName, "webhook-config-name", "", "Name of the MutatingWebhookConfiguration to be configured by osm-controller")
 	flags.StringVar(&osmConfigMapName, "osm-configmap-name", "osm-config", "Name of the OSM ConfigMap")
+	flags.IntVar(&maxEnvoyConnect, "max-envoy-connect", 0, "Indicates the max number of Envoy connections allowed, 0 if disabled")
```

Comment:
This doesn't need to be exposed as a command line argument if we can expose it in the ConfigMap, similar to `envoy_log_level`.